src/Button.vue

```diff
@@ -142,7 +126,7 @@ export default {
     },
 
     buttonRadius () {
-      return this.height - this.margin * 2;
+      return this.height - this.margin * 2
```

Comment:
There are a lot of changes where semicolon is gone.
It is quite hard now to unstained what are functional and what are stylistic changes.
You might want to put it back & mb make another PR wit stylistic changes, tnx